packages/integrations/markdoc/src/content-entry-type.ts

```diff
@@ -106,7 +106,7 @@ export async function getContentEntryType({
 import { createGetHeadings, createContentComponent } from '@astrojs/markdoc/runtime';
 ${
 	markdocConfigUrl
-		? `import markdocConfig from ${JSON.stringify(markdocConfigUrl.pathname)};`
+		? `import markdocConfig from ${JSON.stringify(decodeURI(markdocConfigUrl.pathname))};`
```

Comment:
I think it would be safer we call `fileUrlToPath(markdocConfigUrl)` instead, since this makes sure it's a valid fs path.

Reply:
Hello, thank you for taking the time to review my PR. 
I've updated the code to use the safer API you pointed out!